doc/install/stampede.tex

```diff
@@ -13,32 +13,30 @@
 
 
 
-Setting up ASPECT on TACC Stampede is much easier than on previous systems since
+\noindent Setting up ASPECT on TACC Stampede is much easier than on previous systems since
```

Comment:
I would suggest you simply follow the formatting in the rest of the document, including paragraph indents. Or do you use `\noindent` for a purpose?